packages/scheduled-tasks/src/lib/strategies/ScheduledTaskSQSStrategy.ts

```diff
@@ -0,0 +1,89 @@
+import { container } from '@sapphire/framework';
+import type { SQS } from 'aws-sdk';
+import { randomBytes } from 'crypto';
+import { Consumer, ConsumerOptions } from 'sqs-consumer';
+import { Producer } from 'sqs-producer';
+import type { ScheduledTaskCreateRepeatedTask, ScheduledTasksTaskOptions } from '../types';
+import type { ScheduledTaskBaseStrategy } from '../types/ScheduledTaskBaseStrategy';
+
+export interface ScheduledTaskSQSStrategyMessageBody {
+	task: string;
+	payload?: unknown;
+	options: ScheduledTasksTaskOptions;
+}
+
+export class ScheduledTaskSQSStrategy implements ScheduledTaskBaseStrategy {
+	public readonly options: ConsumerOptions;
+
+	private producer!: Producer;
```

Comment:
If you're going to initialize this later, initialize it as null.
```suggestion
	private producer: Producer | null = null;
```

You can also move the line 25 to the constructor and have it always defined.